mergify_engine/config.py

```diff
@@ -155,6 +159,12 @@ def ApplicationAPIKeys(v: str) -> typing.Dict[str, ApplicationAPIKey]:
         voluptuous.Required("APPLICATION_APIKEYS", default=""): voluptuous.Coerce(
             ApplicationAPIKeys
         ),
+        # Saas Special config
+        voluptuous.Required("ENGINE_TO_DASHBOARD_API_KEY"): str,
+        voluptuous.Required("DASHBOARD_TO_ENGINE_API_KEY"): str,
```

Comment:
Will this block the startup of onprem?

Reply:
I have generated a random secret on startup for onpremise, so that will not block and make this APIs unusable.